Test with PyPI pip-wx job hanging

The `pip-wx` job on the "Test with PyPI" workflow hangs midway through the test run.

Here's the tail end of the log for one such run:
```
2022-06-07T16:48:12.1505593Z test_modify_slider_simple_slider (traitsui.tests.editors.test_range_editor.TestRangeEditor) ... ok
2022-06-07T16:48:12.1507244Z test_range_text_editor_set_with_text_after_empty (traitsui.tests.editors.test_range_editor.TestRangeEditor) ... skipped "Test requires one of these toolkits: [<ToolkitName.qt: 'qt'>]"
2022-06-07T16:48:12.1906355Z test_range_text_editor_set_with_text_valid (traitsui.tests.editors.test_range_editor.TestRangeEditor) ... ok
2022-06-07T22:41:28.9914606Z test_set_text_out_of_range (traitsui.tests.editors.test_range_editor.TestRangeEditor) ... 
2022-06-07T22:41:29.0333612Z ##[error]The operation was canceled.
```

The last time this job passed appears to be this run, 16 months ago: https://github.com/enthought/traitsui/actions/runs/576877114. The following run a week later failed.

--- a/traitsui/tests/editors/test_range_editor.py
+++ b/traitsui/tests/editors/test_range_editor.py
@@ -404,7 +404,9 @@ class TestRangeEditor(BaseTestMixin, unittest.TestCase, UnittestTools):
                 float_value_text.inspect(DisplayedText()), "0.2 ..."
             )
 
-    # regression test for enthought/traitsui#737
+    # regression test for enthought/traitsui#737. Hangs with a popup
+    # dialog on wx. (xref: enthought/traitsui#1901)
+    @requires_toolkit([ToolkitName.qt])
     def test_set_text_out_of_range(self):
         model = RangeModel()
         view = View(

